### PR TITLE
Update to 0.2.20

### PR DESCRIPTION
**Pull request by isuruf**
I've also simplified the build script a bit by using toolchain3. Let me know if you'd like to see that in a new PR and have this PR only update.

### CODE REVIEW

**Comment by jakirkham** [comment]
> Let me know if you'd like to see that in a new PR and have this PR only update.

Would that be alright? Still interested in using `toolchain3`, but it would be nice to make the changes here a little more granular.

**Comment by isuruf** [comment]
Done

File: recipe/build.sh [hunk 1]
```diff
@@ -40,7 +40,9 @@ make install PREFIX="${PREFIX}"
...
 # OpenBLAS. This will make it easier for packages that are looking for them.
-ln -fs $PREFIX/lib/libopenblas.a $PREFIX/lib/libblas.a
-ln -fs $PREFIX/lib/libopenblas.a $PREFIX/lib/liblapack.a
-ln -fs $PREFIX/lib/libopenblas$SHLIB_EXT $PREFIX/lib/libblas$SHLIB_EXT
-ln -fs $PREFIX/lib/libopenblas$SHLIB_EXT $PREFIX/lib/liblapack$SHLIB_EXT
+
+for arg in blas cblas lapack; do
+    ln -fs $PREFIX/lib/pkgconfig/openblas.pc $PREFIX/lib/pkgconfig/$arg.pc
+    ln -fs $PREFIX/lib/libopenblas.a $PREFIX/lib/lib$arg.a
+    ln -fs $PREFIX/lib/libopenblas$SHLIB_EXT $PREFIX/lib/lib$arg$SHLIB_EXT
```
Comment (jakirkham) [inline recipe/build.sh]: Did a similar change go upstream?

File: recipe/build.sh [hunk 2]
```diff
@@ -40,7 +40,9 @@ make install PREFIX="${PREFIX}"
 # As OpenBLAS, now will have all symbols that BLAS or LAPACK have,
 # create libraries with the standard names that are linked back to
 # OpenBLAS. This will make it easier for packages that are looking for them.
-ln -fs $PREFIX/lib/libopenblas.a $PREFIX/lib/libblas.a
-ln -fs $PREFIX/lib/libopenblas.a $PREFIX/lib/liblapack.a
-ln -fs $PREFIX/lib/libopenblas$SHLIB_EXT $PREFIX/lib/libblas$SHLIB_EXT
-ln -fs $PREFIX/lib/libopenblas$SHLIB_EXT $PREFIX/lib/liblapack$SHLIB_EXT
+
```
Comment (jakirkham) [inline recipe/build.sh]: I guess we can trim this blank line as the comment above still seems to apply to this.

**Comment by jakirkham** [comment]
Minor comments/questions. Otherwise LGTM.

File: recipe/build.sh (hunk 1 above)
Comment (jakirkham) [inline recipe/build.sh]: Ok, do you have a link to the discussion?

Comment (isuruf) [inline recipe/build.sh]: https://github.com/xianyi/OpenBLAS/issues/598#issuecomment-278656230

**Pull request merged**